wallet/wallet.go

```diff
@@ -1437,60 +1369,15 @@ func (w *Wallet) TxRecord(txSha *wire.ShaHash) (r *txstore.TxRecord, ok bool) {
 	return nil, false
 }
 
-// openWallet opens a wallet from disk.
-func openWallet() (*Wallet, error) {
-	netdir := networkDir(cfg.DataDir, activeNet.Params)
-	dbPath := filepath.Join(netdir, walletDbName)
-
-	// Ensure that the network directory exists.
-	if err := checkCreateDir(netdir); err != nil {
-		return nil, err
-	}
-
-	// Open the database using the boltdb backend.
-	db, err := walletdb.Open("bdb", dbPath)
-	if err != nil {
-		return nil, err
-	}
-
-	// Get the namespace for the address manager.
-	namespace, err := db.Namespace(waddrmgrNamespaceKey)
-	if err != nil {
-		return nil, err
-	}
-
-	// Open address manager and transaction store.
-	var txs *txstore.Store
-	config := &waddrmgr.Options{
-		ObtainSeed:        promptSeed,
-		ObtainPrivatePass: promptPrivPassPhrase,
-	}
-	mgr, err := waddrmgr.Open(namespace, []byte(cfg.WalletPass),
-		activeNet.Params, config)
-	if err == nil {
-		txs, err = txstore.OpenDir(netdir)
-	}
-	if err != nil {
-		// Special case: if the address manager was successfully read
-		// (mgr != nil) but the transaction store was not, create a
-		// new txstore and write it out to disk.  Write an unsynced
-		// manager back to disk so on future opens, the empty txstore
-		// is not considered fully synced.
-		if mgr == nil {
-			return nil, err
-		}
+// Returns wallet db being used by a wallet
+func (w *Wallet) GetDb() walletdb.DB {
```

Comment:
Please name this `.Db()` per [Effective Go Guidelines](http://golang.org/doc/effective_go.html#Getters).